tests/v1/kv_connector/unit/test_nixl_connector.py

```diff
@@ -378,10 +378,14 @@ def test_concurrent_load_kv(
         raise TimeoutError("Took too long to complete async handshake.")
 
 
+# NOTE: resource cleanup in mp backend is a bit finicky, so the order in which
+# we put here is important. First run ray, it will clean up the resources, then
+# run mp tests.
+@pytest.mark.parametrize("distributed_executor_backend", ["ray", "mp"])
```

Comment:
increasing the test coverage. This test fails on master which means the test is meaningful :)  So it's good that it passes now cc @robertgshaw2-redhat